src/Enum/StatusEnum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Lle\HermesBundle\Enum;
+
+/**
+ * Class StatusEnum
+ * @package Lle\HermesBundle\Enum
+ *
+ * @author 2LE <2le@2le.net>
+ */
+class StatusEnum extends Enum
```

Comment:
je ne veux pas d'ENUM !!!

Reply:
We should always use enums when a variable (especially a method parameter) can only take one out of a small set of possible values.